test/SparqlEndpointFetcher-test.ts

```diff
@@ -303,6 +303,18 @@ describe('SparqlEndpointFetcher', () => {
         );
       });
 
+      it('should use HTTP GET when url length is less than maxUrlLengthForGet', async() => {
```

Comment:
Let's also add a unit test when setting `maxUrlLengthForGet` to 20 or something to show that a query will use POST instead.